buildspec-deploy.yml

```diff
@@ -0,0 +1,29 @@
+version: 0.2
+
+phases:
+    install:
+      runtime-versions:
+        nodejs: 14
+        python: 3.8
+      commands:
+        - env
+        - npm install -g yarn
+        - pip3 install --upgrade pip
+        - pip3 install --upgrade awscli
+        - apt install jq -y
+    build:
+      commands:
+        - ls -al
+        - tar -xzf fhirworks.tgz
+        - ( cd $CODEBUILD_SRC_DIR_DepCog && cat cognito-deploy-output.json )
```

Comment:
Just `cat "$CODEBUILD_SRC_DIR_DepCog/cognito-deploy-output.json"`

Also in other places, don't `cd` just to get to the file, just use the full path.

Reply:
Agree with the cat, with the cd in order to call ./install.sh or yarn install the current path needs to be preserved in that folder because of relative navigation.